colossalai/zero/gemini/gemini_ddp.py

```diff
@@ -17,13 +17,26 @@
 from colossalai.tensor.colo_parameter import ColoParameter
 from colossalai.tensor.param_op_hook import ColoParamOpHookManager
 from colossalai.utils import _cast_float, free_storage, get_current_device, is_ddp_ignored
+from colossalai.checkpoint_io.utils import gather_distributed_param
```

Comment:
Please make sure that `self.dp_process_group` in GeminiDDP is actually the dp group rather than default process group.

Reply:
done